packages/tools/fetch-tool/src/fluidFetchArgs.ts

```diff
@@ -119,6 +116,10 @@ export function parseArguments() {
             case "--dump:snapshotTree":
                 dumpSnapshotTrees = true;
                 break;
+            case "--dump:snapshotTree:overwrite":
```

Comment:
We don't need this. This is for fetching snapshot tree and has nothing to do with messages.